pygen/pygen_src/riscv_asm_program_gen.py

```diff
@@ -125,15 +126,16 @@ def gen_program(self):
                 if not cfg.no_data_page:
                     self.gen_data_page(hart)
 
-                    if((hart == 0) and ("RV32A" in rcs.supported_isa)):
+                    if((hart == 0) and (riscv_instr_group_t.RV32A
+                                         in rcs.supported_isa)):
```

Comment:
nit: redundant parenthesis: 
`if(hart == 0 and riscv_instr_group_t.RV32A
                                         in rcs.supported_isa):`